Tests/AuthServiceTests/UnitTest1.cs

```diff
@@ -0,0 +1,10 @@
+using NUnit.Framework;
+
+namespace AuthServiceTests
+{
+    public class Tests
```

Comment:
We'll use different classes with long names for each testing option, won't we?

Reply:
Agree